src/Page.js

```diff
@@ -1059,8 +1058,6 @@ class Page {
         .then(result => markbinder.resolveBaseUrl(result, {
           baseUrlMap: this.baseUrlMap,
           rootPath: this.rootPath,
-          isDynamic: true,
```

Comment:
these variables in `resolveDependency` are not necessary, as in the `resolveBaseUrl` process, `attribs[ATTRIB_CWF] = context.cwf` is the same assigned to `parser.dynamicIncludeSrc.push({ from: context.cwf, ... })`